src/seedu/addressbook/AddressBook.java

```diff
@@ -398,6 +406,18 @@ private static String executeCommand(String userInputString) {
         return split.length == 2 ? split : new String[] { split[0] , "" }; // else case: no parameters
     }
 
+    /**
+     * Splits update command arguments string into index and person data
+     *
+     * @return  size 2 array; first element is the index and second element is the rest of the
+     * arguments string containing the person data
+     */
+    private static String[] splitIndexAndPersonData(String commandArgs) {
```

Comment:
This method has the same implementation as _splitCommandWordAndArgs_, but I why you would do so based on how you are using it subsequently. It might be better for you to rename the method previous method, and also change the description of the method.